### PR TITLE
[candi] migrate to containerd 1.6

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>

It seems that we should wait for https://github.com/containerd/containerd/pull/7298 to be merged !!!!

## Description

Upgrade containerd version to 1.6 and remove unneeded containerd-flant-edition binary.

## Why do we need it, and what problem does it solve?

New containerd version contains many fixes.
## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: chore
summary: migrate to containerd 1.6
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @RomanenkoDenys. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @konstantin-axenov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @konstantin-axenov. I've started the workflow for label 'e2e/run/gcp'...

**Review by konstantin-axenov — CHANGES_REQUESTED** [review]

File: candi/bashible/bundles/centos/node-group/031_install_containerd.sh.tpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Flant JSC
+# Copyright 2022 Flant JSC
```
[the remark this answers is not in the document]
Comment (RomanenkoDenys) [reply candi/bashible/bundles/centos/node-group/031_install_containerd.sh.tpl]: ok, fixed

File: candi/bashible/common-steps/node-group/032_remove_containerd_fe.sh.tpl [hunk 2]
```diff
@@ -1,5 +1,4 @@
-#!/bin/bash
-# Copyright 2021 Flant JSC
+# Copyright 2022 Flant JSC
```
Comment (konstantin-axenov) [inline candi/bashible/common-steps/node-group/032_remove_containerd_fe.sh.tpl:1]:
```suggestion
# Copyright 2021 Flant JSC
```

**Review by konstantin-axenov — CHANGES_REQUESTED** [review]

File: modules/007-registrypackages/images/containerd-fe/werf.inc.yaml [hunk 3]
```diff
@@ -1,48 +0,0 @@
...
-  - apt-get update
-  - apt-get install -y patch libbtrfs-dev
-  install:
-  - mkdir -p /src
-  - cd /src
-  - git clone https://github.com/containerd/containerd.git
-  - cd containerd
-  - git checkout tags/v{{ $version }}
-  - git config --global user.email "builder@deckhouse.io"
-  - git cherry-pick 5f3ce9512b74718ac9519196ba70dfb1ee075fbe # Do not append []string{""} to command to preserve Docker and Werf compatibility
```
Comment (konstantin-axenov) [inline modules/007-registrypackages/images/containerd-fe/werf.inc.yaml:44]: This is still not in release of containerd - https://github.com/containerd/containerd/blob/release/1.6/pkg/cri/opts/spec.go#L66

Comment (konstantin-axenov) [reply modules/007-registrypackages/images/containerd-fe/werf.inc.yaml:44]: It seem that we should wait for https://github.com/containerd/containerd/pull/7298 to be merged.

**Pull request closed without merging**